hnn_core/tests/test_parallel_backends.py

```diff
@@ -43,6 +43,17 @@ def run_hnn_core(backend=None, n_procs=None, n_jobs=1, reduced=False):
     return dpls, net
 
 
+# Travis CI should fail if it cannot import mpi4py. Otherwise tests below
+# would be skipped and will not trigger a CI failure.
+def test_mpi4py_needed():
+    try:
+        import mpi4py
+        assert hasattr(mpi4py, '__version__')
+    except:
+        if "TRAVIS_OS_NAME" in environ:
+            pytest.fail("Travis CI requires mpi4py to test MPIBackend")
```

Comment:
if you turn this into a decorator, we can force the tests we want to fail instead of another test failing ...

Reply:
@jasmainak feel free to make that change. I won't have time to update this PR.

Reply:
I'll take care of it, thanks for letting me know :)